deploy/operator/setup_assisted_operator.sh

```diff
@@ -172,10 +190,30 @@ EOCR
 }
 
 function from_index_image() {
+  export OPENSHIFT_VERSIONS=$(echo ${OPENSHIFT_VERSIONS} |
```

Comment:
remove as it happens in the `deploy.sh` script

Reply:
done